src/lib/components/Collapsible.svelte

```diff
@@ -59,7 +60,7 @@
 </script>
 
 <div
```

Comment:
It's a good point. Yes, it might be something useful. I agree we shuold wrap it all with a div.

What I'm not so sure is whether this is a concern for gix-components or instead it should be done at the nns-dapp level.

We never discussed this, but coupling gix-components with nns-dapp is not the point of this library. I'm not even sure test ids should be here at all.

Instead, we might want to wrap those components in the nns-dapp level adding a test id.

Reply:
done

Reply:
Those are good questions that I cannot answer. If you prefer to do it on the nns-dapp side, that's fine with me but how would that be possible with gix components which are nested inside other gix components?